src/cfnlint/rules/resources/ApproachingLimitNumber.py

```diff
@@ -17,13 +17,9 @@ class LimitNumber(CloudFormationLintRule):
 
     def match(self, cfn):
         """Check CloudFormation Resources"""
-
         matches = []
-
-        # Check number of resources against the defined limit
         resources = cfn.get_resources()
```

Comment:
wary about switching these from [`get_resources()`](https://github.com/aws-cloudformation/cfn-python-lint/blob/b34913e1186edcf23fd4042d2d197b76c762d708/src/cfnlint/template.py#L63) to [`get('Resources', {})`](https://github.com/aws-cloudformation/cfn-python-lint/blob/b34913e1186edcf23fd4042d2d197b76c762d708/src/cfnlint/decode/node.py#L108) like the rest of these rules